modules/standard/IO.chpl

```diff
@@ -1526,7 +1529,10 @@ proc _modestring(mode:iomode) {
     when iomode.rw do return _rw;
     when iomode.cw do return _cw;
     when iomode.cwr do return _cwr;
-    otherwise halt("Invalid mode");
+    otherwise {
+      use HaltWrappers;
+      exhaustiveSelectHalt("Invalid iomode");
```

Comment:
In https://github.com/chapel-lang/chapel/pull/10327 we switched over to fully qualified names, which I think is a nice cleanup. Switch to 
```chpl 
otherwise do HaltWrappers.exhaustiveSelectHalt("Invalid iomode");
```
?